apps/voting/app/src/components/VoteDetail/DisputableActions.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Button, GU, textStyle, useTheme } from '@aragon/ui'
+import {
+  VOTE_STATUS_PAUSED,
+  VOTE_STATUS_ACTIVE,
+  VOTE_STATUS_CANCELLED,
+  VOTE_STATUS_CLOSED,
+} from '../../disputable-vote-statuses'
+import { getDisputableVoteStatus } from '../../disputable-utils'
```

Comment:
I don't think we need this anymore, and since it's been removed it might be causing an import resolution error.